apps/docs/next.config.js

```diff
@@ -85,6 +85,22 @@ const nextConfig = {
 				destination: '/reference/:slug*',
 				permanent: true,
 			},
+			// redirects at the top level
+			{
+				source: '/examples',
+				destination: '/examples/basic/basic',
+				permanent: true,
+			},
+			{
+				source: '/reference',
```

Comment:
hm i see for `/reference/` and `/docs/` (they 404) we don't have the same redirect like `/examples/` - i'd have to dig into why they don't have the same behavior